tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -3424,6 +3427,7 @@ node_modules/chrome-devtools-frontend/front_end/cm_headless/headlesscodemirror.j
   Types of property 'prototype' are incompatible.
     Type '{ eol: () => boolean; sol: () => boolean; peek: () => any; next: () => any; eat: (match: any) => any; eatWhile: (match: any) => boolean; eatSpace: () => boolean; skipToEnd: typeof skipToEnd; skipTo: typeof skipTo; ... 6 more ...; lookAhead: () => any; }' is not assignable to type '{ backUp: (n: any) => void; column: () => void; current: () => void; eat: (match: any) => void; eatSpace: () => void; eatWhile: (match: any) => void; eol: () => void; indentation: () => void; match: (pattern: string | RegExp, consume?: boolean, caseInsensitive?: boolean) => void; ... 4 more ...; sol: () => void; }'.
       Property 'hideFirstChars' does not exist on type '{ backUp: (n: any) => void; column: () => void; current: () => void; eat: (match: any) => void; eatSpace: () => void; eatWhile: (match: any) => void; eol: () => void; indentation: () => void; match: (pattern: string | RegExp, consume?: boolean, caseInsensitive?: boolean) => void; ... 4 more ...; sol: () => void; }'.
+node_modules/chrome-devtools-frontend/front_end/cm_headless/headlesscodemirror.js(76,1): error TS2322: Type '(mode: any, a1: any, a2: any) => any' is not assignable to type '(mode: any) => void'.
```

Comment:
similar to uglify's mutation of a variable, but the cause this time is Closure's way of representing interfaces with a dummy value assignment.